j2v/generation/processor.py

```diff
@@ -46,6 +46,7 @@ def process_json_files(self, json_file_list):
         self.looker_writer.create_view_file(self.generator.views_dimensions_expr)
         self.looker_writer.create_explore_file(self.generator.explore_joins)
         self.sql_writer.print_sql(self.generator.all_fields, self.generator.all_joins)
+        self.sql_writer.print_sql_no_nulls(self.generator.all_non_null_fields, self.generator.all_joins)
```

Comment:
Let's print SQL only 1 time in 1 version. We should be able to control if we want to print / return it with IFNULL checks or without them, that would simplify a code.